node/pkg/admin/tests/proxy_test.go

```diff
@@ -0,0 +1,157 @@
+//nolint:all
+package tests
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"bisonai.com/orakl/node/pkg/admin/proxy"
+	"bisonai.com/orakl/node/pkg/db"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProxyInsert(t *testing.T) {
+	ctx := context.Background()
+	_cleanup, testItems, err := setup(ctx)
```

Comment:
@nick-bisonai I understand that the underscore is added due to the name collision with function in `main_test.go`, however, I believe that rather than using unconventional naming for local variables at many places, it would have been cleaner to rename the function, which is used only once in the file.

Reply:
I'll make an update